### DISCUSSION TITLE
Google Launches Carbon, an Experimental Replacement for C++

### DISCUSSION THREAD

**Discussion by johnwason** [tesseract-robotics/tesseract#808] [General]
Something to keep an eye on:

https://thenewstack.io/google-launches-carbon-an-experimental-replacement-for-c/

**Comment by Levi-Armstrong** [comment]
I was looking at that yesterday. :) It is nice that it can work in tandem with c++ code.

**Reply by johnwason** [reply]
There is also Dlang; I don't know why they never got any traction. Maybe Google will have better luck.
